combat-trainer.lic

```diff
@@ -1577,7 +1583,12 @@ class SpellProcess
     return if game_state.casting
     return unless @training_spells
     return if @training_spells_max_threshold && game_state.npcs.length > @training_spells_max_threshold
-    return if DRStats.mana < @training_spell_mana_threshold
+    if DRStats.mana < @release_cyclic_threshold && @release_cyclic_on_low_mana == true
+      release_cyclics
+      return
```

Comment:
Should this only apply if you actively have a cyclic up?

Are you seeing combat-trainer spam `release cyclic` over and over in the event loop while waiting for your mana to increase above the threshold?

Reply:
release_cyclic method checks if its up or not.  It does not spam. It was tested.  The only time you see it try it more than once is in the seconds after its released *and* you have nothing else to do so it immediately returns to training spells section before everything actually recognizes the release. (Like I was doing sitting in my safe room with no mobs and only two spells to cast.)   In active combat it should release and just move to the next action in queue.